mos_tests/murano/conftest.py

```diff
@@ -338,3 +338,32 @@ def influx(murano, environment, session, pod):
         }
     }
     return murano.create_service(environment, session, post_body)
+
+
+@pytest.fixture
+def dockerhttpd_package(murano_cli):
```

Comment:
You can use parametrization like @pytest.mark.parametrize('package', [('DockerHTTPd',)], indirect=['package']) instead of dockerhttpd_package